pytorch_lightning/loggers/mlflow.py

```diff
@@ -110,6 +111,7 @@ def any_lightning_module_function_or_hook(self):
 
     def __init__(
         self,
+        run_id: Optional[str] = None,
```

Comment:
this should be added at the end, after `artifact_location` in case users relied on specifying arguments positionally

Reply:
Done! Thanks for the comment!